pinot-controller/src/main/java/org/apache/pinot/controller/api/resources/PinotRealtimeTableResource.java

```diff
@@ -90,6 +90,23 @@ public Response resumeConsumption(
     }
   }
 
+  @POST
+  @Path("/tables/{tableName}/resetConsumption")
+  @ApiOperation(value = "Reset consumption of a realtime table",
+      notes = "Reset the consumption for a realtime table using force commit")
```

Comment:
Maybe add some note in the description about it being async, and 200 response may not mean it has actually been done already? Will there be a followup to this to know status of resetConsumption (like the recent reload status API)?

Reply:
Done.

Reply:
For the status of forceCommit, I can't think of an easy way to figure out if a segment is committed after force commit!